risc0/zkvm/verify/Cargo.toml

```diff
@@ -1,23 +1,34 @@
 [package]
 name = "risc0-zkvm-verify"
-version = "0.1.0"
+version = "0.2.0"
 edition = "2021"
 description = "RISC Zero zero-knowledge VM verify crate"
 license = "Apache-2.0"
 homepage = "https://risczero.com/"
 repository = "https://github.com/risc0/risc0/"
+links = "risc0-zkvm-verify"
+
+[features]
+default = ["pure", "std"]
+cxx = ["dep:cxx", "dep:risc0-core", "dep:risc0-zkvm-circuit", "risc0-zkp-core/cxx"]
+pure = ["dep:risc0-zkp-verify", "dep:risc0-zkvm-serde", "dep:serde"]
```

Comment:
Do we need to put e.g. "risc0-zkp-verify/pure" in these features lines in order for the "pure" feature to propagate to the dependencies?  I'm still a bit fuzzy on cargo dependency specifications.

Reply:
It's new for me as well. Maybe? I haven't run into any issues yet. But my understanding is that because the `pure` feature is in the `default` feature of `risc0-zkp-verify`, then it should be OK to elide it here. But maybe it's better to be explicit anyways.

I'm actually puzzled why I didn't need to put `risc0-zkp-verify/cxx` in the `cxx` feature.

Reply:
OK, I see what's going on. The `cxx` feature doesn't depend on `risc0-zkp-verify` apparently.